scripts/docker_run

```diff
@@ -17,6 +17,7 @@ source "$SCRIPTS_DIR/common"
 readonly DOCKER_UID="${UID:-0}"
 readonly DOCKER_GID="${GID:-0}"
 readonly DOCKER_USER="${USER:-root}"
+readonly DOCKER_GROUP="$(getent group docker | cut -d: -f3)"
```

Comment:
AFAICT this is literally the group of the `docker` user, but the other `DOCKER_*` vars actually refer to the user and group of the user outside of docker, so that it can be remapped inside. Not sure it's worth renaming things, but at least please make it clear in a comment, I can definitely see myself trying to figure this out again in the future :)

Reply:
It is the outside docker group id. Yeah, naming is confusing, I'll change it.